src/cli/cmds/clone.js

```diff
@@ -0,0 +1,35 @@
+import buildBlueprintCommands from './clone/build-blueprint-commands';
+
+const usage = `Usage:
+  $0 clone <blueprint> <name>
+  $0 help clone <blueprint>`;
+
+module.exports = {
+  command: 'clone <blueprint> <name>',
+  aliases: ['g', 'gen'],
+  describe: 'Clone a blueprint with a different name',
+  builder: yargs => {
+    yargs
+      .usage(usage)
+      .option('dry-run', {
+        alias: 'd',
+        describe: "List files but don't generate them",
```

Comment:
@anithri 

Prettier would use the two different quoting styles, because the describe string has an embedded single quote. Eslint should already allow that (and complain if double quotes are used in any other circumstance). The rule for quoting was modified by a recent PR

`"quotes": [2, "single", {"avoidEscape": true}],`

I prefer the prettier formatting for switch, and not having any manual step to undo it! This can also be accommodated by eslint:

`"indent": [2, 2, {"SwitchCase": 1}],`

Suggest we make that change in `.eslintrc`